examples/hosts/app-integration/schema-upgrade/src/migrationTool/migrationTool.ts

```diff
@@ -45,15 +44,23 @@ export class MigrationTool extends DataObject implements IMigrationTool {
         return this._taskManager;
     }
 
-    public get migrated() {
-        return this.crc.read(newContainerIdKey) !== undefined;
+    public get migrationState() {
+        if (this.newContainerId !== undefined) {
+            return "migrated";
+        } else if (this.acceptedVersion !== undefined) {
+            return "migrating";
+        } else if (this.proposedVersion !== undefined) {
+            return "stopping";
+        } else {
+            return "collaborating";
```

Comment:
Maybe add a note for what `"collaborating"` means (the other states feel more self-explanatory)

Reply:
Added documentation for MigrationState in latest